### PR TITLE
Expand owners franchise

### PR DESCRIPTION
**Pull request by calebamiles**
Updates owners files using git commit history. Updates were produced using the same tooling as in https://github.com/kubernetes/kubernetes/pull/31752. PTAL, @grodrigues3.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/contrib/1842)

### CODE REVIEW

File: rescheduler/vendor/k8s.io/kubernetes/pkg/api/OWNERS [hunk 1]
```diff
@@ -1,6 +1,6 @@
-assignees:
```
Comment (eparis) [inline rescheduler/vendor/k8s.io/kubernetes/pkg/api/OWNERS]: no touching stuff in `vendor`

**Comment by calebamiles** [comment]
@grodrigues3, updated PR for your review. Thanks!

**Comment by grodrigues3** [comment]
awesome looks like the right structure to me.  are we just going to assume that people opt-in to this one (since it's a lower time commitment) or are you going to send emails out as well?

**Comment by eparis** [comment]
There are certainly people who got opted into lots of things they likely don't care about. I noticed @foxish and @ingvagabund as the sole reviewers on a whole slew of things...

**Comment by calebamiles** [comment]
I'm happy to send out opt in emails, @grodrigues3 + @eparis

File: cluster-autoscaler/config/OWNERS [hunk 2]
```diff
@@ -0,0 +1,4 @@
+reviewers:
+- ingvagabund
```
Comment (ingvagabund) [inline cluster-autoscaler/config/OWNERS]: I just ask some questions about updated README file. Don't think I can do anything else with the autoscaler in the time given.

**Comment by ingvagabund** [comment]
@calebamiles I am happy to help, though I have only some spare time to share.

**Comment by ide** [comment]
Have you heard of https://github.com/facebook/mention-bot? It's not totally the same but is similar in spirit to the owners file and uses git blame on each PR to infer who the best reviewers might be.

**Comment by foxish** [comment]
@ide, there was https://github.com/kubernetes/contrib/issues/1250 where this was investigated.

**Comment by calebamiles** [comment]
emails have been sent, @grodrigues3 + @eparis. Configuration for the findowner tool will be stored at https://github.com/calebamiles/kubernetes-contrib-reviewers-configuration until a better place can be identified

File: apparmor/OWNERS [hunk 3]
```diff
@@ -0,0 +1,3 @@
+reviewers:
+- ingvagabund
```
Comment (calebamiles) [inline apparmor/OWNERS]: @timothysc, as far as I understood manually opting in reviewers was out of scope for the current work. I'm happy to work on adding that ability if you think it's necessary. I'd also love if @dchen1107 replied to my robot email to signal preferences for reviewing kubernetes/kubernetes :smile:. Also several people have communicated preference to review paths not identified by examining commits so we probably need to have a broader discussion about how to get added to an OWNERS file outside of commit history

**Comment by wernight** [comment]
Looking fine to add me, just possibly include me in the Nginx Ingress Controller as well.

**Comment by grodrigues3** [comment]
I accept

**Comment by aledbf** [comment]
I accept

**Comment by gregory-lyons** [comment]
I accept

**Comment by simonswine** [comment]
I accept

**Comment by ryane** [comment]
I accept

**Comment by grodrigues3** [comment]
@calebamiles when you rebase, feel free to write over the OWNERs file I added to mungegithub/issue-labeler/ .  I was trying to test and see if I could get /approve to work

**Comment by lavalamp** [comment]
/approve

I am not sure what the status of this is, but here is a preemptive approval for when @calebamiles and/or @grodrigues3 pick it up again.

**Pull request closed without merging**